sdk/core/core-amqp/.eslintrc.json

```diff
@@ -1,7 +1,9 @@
 {
   "plugins": ["@azure/azure-sdk"],
-  "extends": ["../../.eslintrc.old.json", "plugin:@azure/azure-sdk/recommended"],
-  "parserOptions": {
-    "createDefaultProgram": true
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"],
+  "rules": {
+    "@typescript-eslint/no-unused-vars": "off",
```

Comment:
Isn't our TSConfig set up to disallow unused variables already? Do we even need this rule at all?

Reply:
Yes! I did this because of some warnings, but it should be safe to remove it. I will try later.